po/fr_FR.UTF-8.po

```diff
@@ -0,0 +1,158 @@
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: lutris\n"
+"Report-Msgid-Bugs-To: \n"
```

Comment:
Missing

Reply:
The first two lines are necessary for the execution of `ninja lutris-update-po`.